src/components/organisms/connections/integrations/kubernetes/edit.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
+
+import { useWatch } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+
+import { integrationVariablesMapping } from "@src/constants";
+import { useConnectionForm } from "@src/hooks";
+import { setFormValues } from "@src/utilities";
+import { kubernetesIntegrationSchema } from "@validations";
+
+import { Button, ErrorMessage, Spinner, Textarea } from "@components/atoms";
+
+import { FloppyDiskIcon } from "@assets/image/icons";
+
+export const KubernetesIntegrationEditForm = () => {
+	const { t } = useTranslation("integrations");
+
+	const { connectionVariables, control, errors, handleSubmit, isLoading, onSubmitEdit, register, setValue } =
+		useConnectionForm(kubernetesIntegrationSchema, "edit");
+
+	const key = useWatch({ control, name: "config_file" });
+
+	useEffect(() => {
+		setFormValues(connectionVariables, integrationVariablesMapping.kubernetes, setValue);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [connectionVariables]);
+
+	return (
+		<form className="flex flex-col gap-4" onSubmit={handleSubmit(onSubmitEdit)}>
+			<div className="relative">
+				<Textarea
+					rows={5}
+					{...register("config_file")}
+					aria-label={t("kubernetes.placeholders.configFile")}
+					disabled={isLoading}
+					isError={!!errors.config_file}
+					placeholder={t("kubernetes.placeholders.configFile")}
+					value={key}
+				/>
+				<ErrorMessage>{errors.config_file?.message as string}</ErrorMessage>
+			</div>
+
+			<Button

```

Comment:
@haimzlato should we add additional info, like here:
![image](https://github.com/user-attachments/assets/349e2e06-5a73-49d3-9b64-08372c29f01c)